src/Serilog.Sinks.RabbitMQ/Sinks/RabbitMQ/RabbitMQClientConfiguration.cs

```diff
@@ -20,10 +20,9 @@ namespace Serilog.Sinks.RabbitMQ.Sinks.RabbitMQ
     /// <summary>
     /// Configuration class for RabbitMqClient
     /// </summary>
-    public class RabbitMQConfiguration
+    public class RabbitMQClientConfiguration
     {
-        public string Hostname { get; set; } = string.Empty;
-        public IList<string> Hostnames { get; } = new List<string>();
+        public IList<string> Hostnames { get; set; } = new List<string>();
```

Comment:
I've tested that both `IConfiguration.GetSection("").Get<RabbitMQClientConfiguration>()` and  `IConfiguration.GetSection("").Bind(config)` works with readonly collection.

This line would thus be `public IList<string> Hostnames { get; } = new List<string>();`

Reply:
Thanks for testing it out. I will update the Hostnames property.